presto-main/src/main/java/io/prestosql/sql/planner/plan/AggregationNode.java

```diff
@@ -204,6 +205,13 @@ public PlanNode replaceChildren(List<PlanNode> newChildren)
         return new AggregationNode(getId(), Iterables.getOnlyElement(newChildren), aggregations, groupingSets, preGroupedSymbols, step, hashSymbol, groupIdSymbol);
     }
 
+    public boolean isDistinct()
```

Comment:
@findepi @martint How do you find this name?  It conflicts a bit with `hasDistinct` in https://github.com/prestosql/presto/pull/551/files#diff-4d5431d3a01d3972523f415074a8d1c5R222

Reply:
I’d probably call this “producesDistinctRows”. Aggregations are not inherently “distinct”, so the name is not appropriate.

In the future, we’ll want to model this as a trait.